src/utils/log/FeatureLog.ts

```diff
@@ -24,7 +25,8 @@ const FeatureLogFlag: TFeatureLogFlag = {
   useOneSignalSubscribeToCommunityHooks: false,
   oneSignalUtils: false,
   navigation: false,
-  firebaseUtils: false
+  firebaseUtils: false,
+  useSystemMessage: false
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding `useSystemMessage` with a default value of `false` to the `FeatureLogFlag` object is consistent with the rest of the codebase. This change is well done.
```diff
+  firebaseUtils: false,
+  useSystemMessage: false
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->